epregressions/energyplus.py

```diff
@@ -189,13 +205,22 @@ def execute_energyplus(e_args: ExecutionArguments):
                 readvars + ' in.mvi', shell=True, stdin=subprocess.DEVNULL,
                 stdout=subprocess.PIPE, stderr=subprocess.PIPE
             )
-        mtr_run.communicate()
+        o, e = mtr_run.communicate()
+        std_out += o
+        std_err += e
+
+        if len(std_out) > 0:
+            with open('eplusout.stdout', 'w') as f:
+                f.write(std_out.decode('utf-8'))
+        if len(std_err) > 0:
+            with open('eplusout.stderr', 'w') as f:
+                f.write(std_err.decode('utf-8'))
 
         os.remove(new_idd_path)
         return [e_args.build_tree['build_dir'], e_args.entry_name, True, False]
 
-    except Exception:
-        return [e_args.build_tree['build_dir'], e_args.entry_name, False, False]
+    except Exception as e:
+        return [e_args.build_tree['build_dir'], e_args.entry_name, False, False, str(e)]
```

Comment:
Added an optional new argument to the return object that can be used to report error messages.  Should probably flesh this out a bit more later.